docs/BMI-Installation.md

```diff
@@ -0,0 +1,240 @@
+# BMI
+
+
+## Installation
+
+This document describes how to setup BMI. We officially support RHEL and it's
+variants, but it may also work on ubuntu (Debian) like OSes. 
+
+### Prerequisite Softwares
+
+BMI requires the following non python dependencies. 
+
+#### Ceph Client
+Ceph is a distributed object store and file system designed for performance and scalibility. 
+For more information, visit the [official website](http://docs.ceph.com/docs/master/start/)
+
+To install ceph client on RHEL:
+```
+yum install ceph-common
+```
+
+To install ceph client on ubuntu:
+```
+apt-get install ceph-common
```

Comment:
Remove the ubuntu related commands. We said we officially support RHEL, so no need for this. Also iSCSI section doesnt have an ubuntu section so better remove it.